content/docs/command-reference/remote/modify.md

```diff
@@ -609,6 +624,17 @@ more details.
 
 > Please note our [Privacy Policy (Google APIs)](/doc/user-guide/privacy).
 
+- `gdrive_acknowledge_abuse` - acknowledge the risk of downloading potential
+  malware. Files identified as
+  [abusive](https://support.google.com/docs/answer/148505) (malware, etc.) are
+  only downloadable by the owner. Consider using this option if you see a
+  `This file has been identified as malware or spam and cannot be downloaded`
+  error.
```

Comment:
```suggestion
- `gdrive_acknowledge_abuse` - acknowledge the risk of downloading potentially
  [abusive](https://support.google.com/docs/answer/148505). Files identified
  as such (malware, personal info., etc.) can only be downloaded by their
  owner.
```

Reply:
> Consider using this option if you see a
  `This file has been identified as malware or spam and cannot be downloaded`
  error.

We should add a [troubleshooting](https://dvc.org/doc/user-guide/troubleshooting) entry instead, I think.

Reply:
> let's keep this part: "Consider using this option if you see ..." it's important for search, etc

@shcheklein my suggestion was to make a troubleshooting entry instead ☝🏼

WDYT @efiop ? I could also keep it for now and make an issue to move it to Troubleshooting later, lmk.

Reply:
I think it's fine to keep the message here for now. It'll be searchable that's the most important thing.